File: Pemrograman 1/MODUL 2/Bahasa C/PRAK204-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>
#define PI 22.0/7.0

int main() {
    float jari_jari, tinggi, volume, luas, keliling;
    
    printf("");
    scanf("%f", &jari_jari);
    printf("");
    scanf("%f", &tinggi);
    
    volume = PI * jari_jari * jari_jari * tinggi;
    luas = 2 * PI * jari_jari * (jari_jari + tinggi);
    keliling = 2 * PI * jari_jari;
    
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);
    
    return 0;
}

File: Pemrograman 1/MODUL 2/Bahasa Python/PRAK204-2410817110002-Akhmad_Daffa_Azzikri.py
PI = 22.0 / 7.0
print("")
inputs = input().split()

if len(inputs) == 2:
    jari_jari = float(inputs[0])
    tinggi = float(inputs[1])
else:
    jari_jari = float(inputs[0])
    tinggi = float(input(""))

volume = PI * jari_jari * jari_jari * tinggi
luas = 2 * PI * jari_jari * (jari_jari + tinggi)
keliling = 2 * PI * jari_jari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")